settings.gradle

```diff
@@ -61,9 +61,12 @@ include 'TrafficCapture:trafficCaptureProxyServer'
 include 'TrafficCapture:trafficReplayer'
 include 'frontend'
 include 'buildImages'
+includeBuild('TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link')
```

Comment:
Are you aware of any limitations with using composite builds here? I'm wondering if we can avoid this by having our sync task look like this: 
```
tasks.register('syncBackendClientDefinition', Copy) {
    dependsOn ':console_link:fetchOpenApiSpec'

    from project(':console_link').layout.buildDirectory.file('openapi.json')
    into project(':frontend').layout.buildDirectory.dir('openapi').get().asFile
}
```